include/benchmark/macros.h

```diff
@@ -27,11 +27,11 @@
 #if defined(__GNUC__)
 # define BENCHMARK_UNUSED __attribute__((unused))
 # define BENCHMARK_ALWAYS_INLINE __attribute__((always_inline))
-# define BENCHMARK_NOEXCEPT noexcept
+# define BENCHMARK_NOEXCEPT(x) noexcept(x)
 #elif defined(_MSC_VER) && !defined(__clang__)
 # define BENCHMARK_UNUSED
 # define BENCHMARK_ALWAYS_INLINE __forceinline
-# define BENCHMARK_NOEXCEPT
+# define BENCHMARK_NOEXCEPT(x)
```

Comment:
This should test _MSC_VER so that it uses conditional noexcept in 2015 or noexcept(false) cases will go to terminate() there.